### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by mdre** [ArcadeData/arcadedb#407] [General]
There's extremely low activity en the repo and a month without commit.  (Last commit: 20220424)

**Comment by lvca** [comment]
Hi, @mdre absolutely not. We're all busy with a client on a migration from OrientDB to ArcadeDB. We should be all back on June the 2nd ;-)

**Reply by fel-mazo** [reply]
hey ! it would be cool if you documented the experience in a blog post or something like that

**Comment by mdre** [comment]
Great! Thanks.

**Comment by lvca** [comment]
Hi guys, from June 2nd we are back 50% on the project. Already merged many PRs, which we will release at the beginning of the next week. Stay tuned.

**Comment by kran** [comment]
Great!! I like this database!
